processor/src/main/java/org/mapstruct/ap/services/DefaultAccessorNamingStrategy.java

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.services;
+
+import java.beans.Introspector;
+
+import org.mapstruct.spi.AccessorNamingStrategy;
+
+/**
+ * The default JavaBeans-compliant implementation of the {@link AccessorNamingStrategy} service provider interface.
+ *
+ * @author Christian Schuster
+ */
+class DefaultAccessorNamingStrategy implements AccessorNamingStrategy {
```

Comment:
I wonder if it would be easier for implementors of the SPI, if we included the default implementation in the SPI module itself (e.g. in a package org.mapstruct.spi.default or something like that). 

Pro: the simple implementations of the methods make it much easier to grasp what the default behaviour is and what the Javadoc on the interface actually means. Plus, if we also made the default implementation ```public``` instead of having a default visibility, people can just extend the default implementation and overwrite only selected methods. That might make future extensions or even changes in the default behaviour easier to adapt for SPI users.

Con: Future default implementations might require more elaborate classes and we might have to break that principle then.

WDYT?